### DISCUSSION TITLE
Bun Support

### DISCUSSION THREAD

**Discussion by slavakurilyak** [directus/directus#22040] [Feature Requests]
### Summary

It would be great to see [Bun](https://bun.sh/) support since it has shown to be more performant than Node.js

<img width="418" alt="Screen Shot 2024-04-01 at 1 03 24 PM" src="https://github.com/directus/directus/assets/6625584/bfe1cde0-1d0e-4cae-80b4-4ff1fbecb816">

Related issue: https://github.com/directus/directus/issues/22039

### Basic Example

```sh
bun create directus-project@latest new-directus-project
cd new-directus-project
bunx directus start
```

### Motivation

It's time to improve the performance of Directus server

### Detailed Design

TBD

### Requirements List

TBD

### Drawbacks

TBD

### Alternatives

N/A

### Adoption Strategy

TBD

**Comment by br41nslug** [comment]
Lets keep the discussion about Bun over here https://github.com/directus/directus/discussions/19643
